IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MemberTypeTieBar/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MemberTypeTieBar" UniqueId="b1fd2844-50d7-3df5-c01e-fa4fa52aaec4" ApplicableType="IfcMember/TIEBAR" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="IsTemporaryInstallation_3I8hgbGqb2cwnJJlcup1s_" />
+		<DocProperty xsi:nil="true" href="TrackEdge_18qZEZZgH0YeAVlHW1LDUp" />
```

Comment:
Please refer to the Track Edge conversations from other pulls

Reply:
TrackEdge is in the BackLog... so it has been erased. Commit : 15b79d25de7519c2d0bf4c1d8a3f413d95e3ee63